interpreter/cling/lib/Interpreter/IncrementalParser.h

```diff
@@ -68,9 +71,6 @@ namespace cling {
     // file ID of the memory buffer
     clang::FileID m_VirtualFileID;
 
-    // CI owns it
```

Comment:
Yeah well only root doesn't ever call the dtor of `DeclCollector` - I did check that before starting to blindly fix this ;-) I.e. there is a problem, Coverity seems right, but the solution is not that easy :-(